test/runner/requirements/integration.cloud.aws.txt

```diff
@@ -1,2 +1,2 @@
 boto
-boto3
+boto3 >= 1.7
```

Comment:
Constraints should be in `constraints.txt`, with a comment explaining why they are needed.

Reply:
Oh, I thought I'd fixed that, I must not have pushed the fix properly. Thanks for the explanation though.